images/rootfs.yml.in

```diff
@@ -72,3 +72,5 @@ services:
 files:
    - path: /etc/eve-release
      contents: 'EVE_VERSION'
+   - path: etc/linuxkit-eve-config.yml
```

Comment:
oh now we ship the file with the image ? Makes sense. BTW, its that a typo ? should be /etc right ?

Reply:
Yeah, that was what I wanted to do. I am writing the source-tracing doc, and realized that making the user run a command to regenerate the rootfs.yml is silly; we should just carry the actual config. To top it all off, linuxkit has supported it for a long time; we can just use it.

> BTW, is that a typo ? should be /etc right?

Nope. It is relative to the root, so it can be with or without. The examples in https://github.com/linuxlkit/linuxkit do it without.